lib/components/molecules/QuoteMedia/QuoteMedia.style.js

```diff
@@ -0,0 +1,23 @@
+import { css } from 'styled-components';
+
+const styles = css`
+  .col-1-2 {
```

Comment:
.col-1-2 could be a global class you might be overwriting, make it quote specific and name it differently